quolance-ui/src/app/(with-main-layout)/(unauthenticated-pages)/whats-new/blog/page.tsx

```diff
@@ -1,13 +1,170 @@
+    profilePicture: '',
+    date: 'November 30, 2024',
+    tags: ['Finance', 'Freelancing', 'Tips']
+  },
+  {
+    type: 'News',
+    title: 'The Role of AI in Backend Automation',
+    body: 'Artificial Intelligence is transforming the way backend systems are managed. This article delves into how AI is used for database optimization, server management, and API handling.',
+    userName: 'AI Innovators',
+    fieldOfWork: 'Backend Development',
+    profilePicture: '',
+    date: 'December 20, 2024',
+    tags: ['Backend', 'AI', 'Automation']
+  },
+  {
+    type: 'Event',
+    title: 'Finance Tools Expo for Freelancers',
+    body: 'Discover the latest finance tools designed specifically for freelancers at our annual Finance Tools Expo. Get insights from industry leaders and explore tools that make financial management a breeze.',
+    userName: 'Freelance Finance',
+    fieldOfWork: 'Finance',
+    profilePicture: '',
+    date: 'February 1, 2025',
+    tags: ['Finance', 'Events', 'Freelancing']
+  },
+  {
+    type: 'Article',
+    title: 'The Best Backend Frameworks for Startups',
+    body: 'Startups often struggle to choose the right backend framework. This article discusses the most efficient and scalable backend frameworks that startups should consider.',
+    userName: 'Tech Startup Guide',
+    fieldOfWork: 'Backend Development',
+    profilePicture: '',
+    date: 'December 8, 2024',
+    tags: ['Backend', 'Development', 'Startups']
+  },
+  {
+    type: 'News',
+    title: 'The Importance of Cybersecurity for Small Businesses',
+    body: 'Small businesses are increasingly targeted by cyberattacks. This article explores the best practices and tools to secure your business from digital threats.',
+    userName: 'CyberSafe Team',
+    fieldOfWork: 'Security',
+    profilePicture: '',
+    date: 'January 5, 2025',
+    tags: ['Security', 'Small Business', 'Tips']
+  },
+  {
+    type: 'Event',
+    title: 'Freelancing in Content Writing: Skills for 2025',
+    body: 'This event will cover the evolving skills required for successful content writing in the freelance market. Learn from top writers and improve your skills for the upcoming year.',
+    userName: 'Content Writers United',
+    fieldOfWork: 'Content Writing',
+    profilePicture: '',
+    date: 'February 20, 2025',
+    tags: ['Freelancing', 'Content Writing', 'Workshops']
+  },
+  {
+    type: 'Article',
+    title: '5 Steps to Improve Your Security Practices',
+    body: 'Whether you are a freelancer or a small business owner, security should be a top priority. This article provides five easy steps to improve your online and offline security practices.',
+    userName: 'Secure Future',
+    fieldOfWork: 'Security',
+    profilePicture: '',
+    date: 'January 2, 2025',
+    tags: ['Security', 'Tips', 'Freelancing']
+  }
+];
+
 
 function BlogPage() {
   return (
     <>
-      <div>
-        Hellow World
+      <div className="bg-gray-800 dark:bg-gray-950">
```

Comment:
here instead of bg-gray-800 you would have something like bg-white and you can remove all occurences of **dark:** as we wont be using that in the future

Reply:
Sounds good

Reply:
![image](https://github.com/user-attachments/assets/33712622-f2cb-4773-932e-6d4eb3d2729a)